### ISSUE TITLE
A dark border?

### ISSUE THREAD

**Issue by Notaphish** [berzniz/github_pr_tree#61]
Hey! Love the plugin and especially dark mode! I've not contributed to someone else's repo before, so forgive me if I've missed something but I tried to raise a branch and PR but I didn't have perms.. guessing it should be raised as an issue first?

Basically, dark mode works great with tampermonkey and the github dark theme, except the border around the tree view! Its white! :P

Before:
<img width="529" alt="Screen Shot 2019-04-11 at 23 31 36" src="https://user-images.githubusercontent.com/1225653/55998466-41d82b80-5cb6-11e9-8c26-c11879c74ceb.png">

After:
<img width="549" alt="Screen Shot 2019-04-11 at 23 39 09" src="https://user-images.githubusercontent.com/1225653/55998473-48ff3980-5cb6-11e9-90eb-f0a24a27c5c7.png">

CSS change:
``` css

.__better_github_pr_dark_mode .__better_github_pr {
    border-color: #484848;
    background-color: #282923;
    color: #ffffff;
}
```

What do you think?  I understand if you want to leave it as is though, as most people might be using the white version of github.

**Comment by berzniz** [comment]
Looks great. I'll add it. You should be able to fork and submit a Pull Request, but if not, I'll add your suggested code.

**Comment by Notaphish** [comment]
Thanks for that, my first open source commit so wasn't sure how it was done! PR is https://github.com/berzniz/github_pr_tree/pull/62

**Comment by berzniz** [comment]
Included in version `1.0.20`. Thank you @Notaphish
